src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -79,44 +104,25 @@ RUN for iter in {1..10}; do \
       {{packageManager}} update  && \
       {{packageManager}} upgrade -y  && \
       {{packageManager}} install -y --no-install-recommends \
-       fontconfig fonts-liberation libnss3 libfontconfig1 ca-certificates curl && \
```

Comment:
The FIPS section still needs `curl`. We could try moving the OpenSSL compile in the builder step, it would allow not having to install `perl make gcc tar` into the image.

Reply:
fips is based on the ubi image right?  I only removed it from the Ubuntu image.

Reply:
Ya, its uses the UBI image. Missed the `ubuntu` tag, sorry.